backend/lib/src/constants/mod.rs

```diff
@@ -24,6 +24,9 @@ pub mod rpc {
     /// Default RPC WebSocket URL
     pub const DEFAULT_RPC_URL: &str = "ws://localhost:9944";
 
+    /// Default MSP callback URL
+    pub const DEFAULT_MSP_CALLBACK_URL: &str = "http://host.docker.internal:8080";
```

Comment:
would probably have localhost here like above

Reply:
I added that as Im using it that way to test manually: 
- I have a MSP node running in a container 
- I run the backend locally 

But I dont have a strong preference